src/backends/ilp-quoter/index.js

```diff
@@ -85,28 +75,40 @@ class ILPQuoter {
                 currencyPair[0] + '/' + currencyPair[1] + '/' + amount +
                 '/' + type
     const result = yield request({ uri, json: true })
-    const fixedAmount = type === 'source' ? result.body.source_amount
-                                          : result.body.destination_amount
+    if (result.statusCode >= 400) {
+      log.error('Error getting quote: ', JSON.stringify(result.body))
+      throw new ServerError('Unable to get quote from backend.')
+    }
+    const source_amount = result.body.source_amount
+    const destination_amount = result.body.destination_amount
+    if (!source_amount || !destination_amount) {
```

Comment:
Should this validation maybe be in models/quote? These validation rules are across any type of backend